doc/getting-started/building-the-node-using-nix.md

```diff
@@ -10,7 +10,7 @@ than installing on Ubuntu/Debian.
 The [Nix Package Manager][nix] can be installed on most Linux distributions by downloading and
 running the installation script:
 ```
-curl https://nixos.org/nix/install > install-nix.sh
+curl -L https://nixos.org/nix/install > install-nix.sh
```

Comment:
We should uses the long version `--location` of the argument just for documentation sake.

Reply:
This is how `nix` documents the install step.  It's probably best to follow their lead on this.

Reply:
![image](https://user-images.githubusercontent.com/63014/91002153-3bc7be00-e611-11ea-98c7-a583078a328d.png)